Clinical-Genomics/scout
# Edit/create panels with genes in build 38

Fix #2726 and fix #1753

How to test:

- On a local instance of Scout containing all genes (37 and 38) edit the demo gene panel with gene NOTCH2NLC (present only with genome build 38).
- Try to load a new gene panel from file after modifying the file to include gene NOTCH2NLC. Tip: replace a gene from this file with 53924 and NOTCH2NLC.
- In both cases the panel should be updated with the gene
- Note that the multiselect containing the options for the disease transcripts now contains more options (tx for both builds and eventual previously selected transcripts). These transcripts may be the same, but I was thinking better safe than sorry..

Expected outcome:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by MHKC
- [x] tests executed by MHKC, CR
